<!--xml-->
<launch>
<node name="$(env ROS_HOSTNAME)_diagnostic_node" pkg="eros" type="diagnostic_node"  output="screen" clear_params="true">
        <param name="startup_delay"       value="0.0"/>
	    <param name="verbosity_level"       value="NOTICE"/>
        <param name="require_pps_to_start"  value="false"/>    
        <param name="loop1_rate"            value="1"/> 
        <param name="loop2_rate"            value="0.1"/>
        <param name="loop3_rate"            value="5"/>
</node>
</launch>
